.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "semi": false,
-  "singleQuote": true
+  "singleQuote": true,
+  "arrowParens": "avoid"
```

Comment:
Ah, I think we'll continue with using the default of this setting and make any changes required as we update prettier in other repos.

Reply:
That was the previous default, it changed with the upgrade, I added it to match the other repos

Reply:
Yes exactly; it went undocumented but we made a decision a while back on prettier v2 to align with the new defaults :).

Reply:
Oh, yes ok!